stream-chat-android-client/src/main/java/io/getstream/chat/android/client/parser/FilterObjectTypeAdapter.kt

```diff
@@ -35,7 +35,7 @@ internal class FilterObjectTypeAdapter(val gson: Gson) : TypeAdapter<FilterObjec
     }
 }
 
-private fun FilterObject.toMap(): Map<*, *> = when (this) {
+internal fun FilterObject.toMap(): Map<*, *> = when (this) {
```

Comment:
Decided to expose this logic within the module instead of duplicating it in the Moshi packages - we can duplicate it later if we want to decouple, but it's probably enough to keep it around for now & move it when the Gson impl is removed